scalameta/trees/shared/src/main/scala/scala/meta/Trees.scala

```diff
@@ -444,6 +448,44 @@ object Member {
   @branch trait ParamClause extends SyntaxValuesClause {
     def values: List[Param]
   }
+
+  @ast class ParamClauseGroup(
+      tparamClause: sm.Type.ParamClause,
+      paramClauses: List[sm.Term.ParamClause]
+  ) extends Tree with Tree.WithTParamClause with Tree.WithParamClauses {
```

Comment:
Do we need `WithParamClauses` still? It's only used here, or is it still used somewhere?

Reply:
in the formatter 🙂 rather than listing a number of various specific trees, i try to group them via a relevant interface.

Reply:
Sure makes sense!